src/api/rest/server/mcis/monitor.go

```diff
@@ -15,7 +15,7 @@ import (
 // RestPostInstallMonitorAgentToMcis godoc
 // @Summary InstallMonitorAgent MCIS
 // @Description InstallMonitorAgent MCIS
-// @Tags MCIS
+// @Tags MONITOR
```

Comment:
@seokho-son 
How about `Monitor` instead of `MONITOR` for Swagger API group name?